DotNet UWP Samples/Earthquake Simulator/Earthquake Simulator.sln

```diff
@@ -0,0 +1,77 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.29409.12
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Earthquake Simulator", "Earthquake Simulator\Earthquake Simulator.csproj", "{77A8F3EB-2C9B-4A82-9F52-55BF4B10DF12}"
```

Comment:
Should this be named "Earthquake workbook Simulator" instead?

Reply:
No, It's simulating an earthquake not a workbook,